### DISCUSSION TITLE
How to best plot general expression data?

### DISCUSSION THREAD

**Discussion by sr320** [RobertsLab/resources#1388] [Q&A] [answered]
I am wanting to graph general characteristics of gene expression in several libraries (snippet below)

Something as simple as stacked bar that bins expression values, violin plot, etc.

for example below I am lookin for something with data plots (one for each library)

| Gene                     |  SRR12207404 |  SRR12207405 |  SRR12207406 |  SRR12207407 |
|--------------------------|--------------|--------------|--------------|--------------|
|                          |              |              |              |              |
| TRINITY_DN8691_c4_g1_i2  | 1230780      | 1.30E+06     | 9.64E+05     | 780843       |
| TRINITY_DN43743_c3_g1_i1 | 747087       | 8.35E+05     | 6.20E+05     | 495716       |
| TRINITY_DN9614_c1_g3_i1  | 607965       | 6.80E+05     | 4.67E+05     | 382833       |
| TRINITY_DN46703_c9_g1_i4 | 543311       | 4.26E+05     | 5.42E+05     | 485921       |
| TRINITY_DN43743_c3_g2_i1 | 348579       | 3.59E+05     | 2.46E+05     | 241063       |
| TRINITY_DN9614_c1_g3_i3  | 323561       | 4.29E+05     | 2.55E+05     | 218356       |
| TRINITY_DN3110_c0_g1_i5  | 268888       | 1.72E+05     | 1.57E+05     | 81328.3      |

maybe like
<img width="169" alt="Latent_effects_of_winter_warming_on_Olympia_oyster_reproduction_and_larval_viability_-_ScienceDirect_2022-02-06_09-45-22" src="https://user-images.githubusercontent.com/4442039/152693961-a2c0a8a2-8517-4c1e-bc21-919b89f09017.png">

**Comment by sr320** [comment, marked as answer]
I have a feeling need to change to data frame were I have a column for `Library`

**Reply by kubu4** [reply]
You might be able to use this as a guide/reference? It would be boxplot/library.

https://github.com/epigeneticstoocean/2018_L18-adult-methylation/blob/main/code/05-ballgown-boxplots.md

**Reply by sr320** [reply]
Thanks - that looks great!

**Reply by kubu4** [reply]
> Thanks - that looks great!

It should! You wrote it!  😄
